packages/esm-ward-app/src/ward-view/ward-view.component.tsx

```diff
@@ -70,6 +70,7 @@ const WardViewByLocation = () => {
         const admittedPatient = admittedPatientsByUuid.get(patient.uuid);
 
         if (admittedPatient) {
+          admittedPatientsByUuid.delete(patient.uuid); // remove the patient from the admittedPatientsByUuid map so at the end we have a list of patients that are admitted but have no beds assigned
```

Comment:
I don't love this.  I'd rather we either have a variable with a different name, or just maintain a Map of patients with beds assigned and then check if the patient is in one but not the other.  I don't love having a Map named `admittedPatientsByUuid` that doesn't actually contain admitted patients by uuid at all times.

Reply:
yeah, i wasn't crazy about it either, but wasn't sure if I was nit-picking, I can probably do some sort of filter.